package.json

```diff
@@ -62,14 +62,15 @@
     "all": "yarn -s install && yarn -s lint && yarn -s build",
     "browser": "yarn -s --cwd examples/browser",
     "build": "yarn -s compile && yarn -s build:examples",
-    "build:examples": "yarn browser build && yarn electron build",
+    "build:examples": "yarn browser build && yarn electron build && yarn browser-only build",
     "clean": "yarn -s rebuild:clean && yarn -s lint:clean && node scripts/run-reverse-topo.js yarn -s clean",
     "compile": "echo Compiling TypeScript sources... && yarn -s compile:clean && yarn -s compile:tsc",
     "compile:clean": "ts-clean dev-packages/* packages/*",
     "compile:references": "node scripts/compile-references.js",
     "compile:tsc": "tsc -b",
     "docs": "rimraf gh-pages/docs/next && typedoc --tsconfig tsconfig.json --options configs/typedoc.json .",
     "download:plugins": "theia download:plugins",
+    "browser-only": "yarn -s --cwd examples/browser-only",
```

Comment:
Minor: We should move this after the `browser` script. I almost overlooked it during testing.